content/en/blog/_posts/2022-TBD-storage-capacity-GA/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Storage Capacity Tracking reaches GA in Kubernetes 1.24"
+date: 2022-12-31
+slug: storage-capacity-ga
+---
+
+ **Authors:** Patrick Ohly (Intel)
+
+With ["storage capacity tracking"](/docs/concepts/storage/storage-capacity/)
+promoted to GA in Kubernetes 1.24, support for local storage in Kubernetes has
+reached its next milestone.
+
+## Problems we are solving
+
+As explained in more detail in the [previous blog post about this
+feature](2021-04-14-local-storage-features-go-beta.md), storage capacity
```

Comment:
The link target is not supposed to be the markdown file. It should be the public HTML file instead, i.e., `/blog/2021/04/14/local-storage-features-go-beta/`.